test/extensions/filters/listener/proxy_protocol/proxy_protocol_test.cc

```diff
@@ -222,6 +222,19 @@ TEST_P(ProxyProtocolTest, V1Basic) {
   disconnect();
 }
 
+TEST_P(ProxyProtocolTest, DetectNoProxyProtocol) {
```

Comment:
Please add a comment on what the test is validating.

Reply:
Addressed in https://github.com/envoyproxy/envoy/pull/18951/commits/4259f1ae4d2bef34b4b80daa48fb4f3516165ad7, please let me know if this doesn't clarify what might have been confusing